test/fixtures/__tests__/scenarioManager.js

```diff
@@ -0,0 +1,83 @@
+var expect = require('expect.js');
+var sinon = require('sinon');
+var Promise = require('bluebird');
+
+var ScenarioManager = require('../scenarioManager');
+
+describe('scenario manager', function () {
+  var manager = new ScenarioManager('http://localhost:9200');
+
+  describe('loading and unloading', function () {
+    this.timeout(60000);
+
+    var bulk;
+    var create;
+    var indicesDelete;
+    beforeEach(function () {
+      bulk = sinon.stub(manager.client, 'bulk', Promise.resolve);
+      create = sinon.stub(manager.client.indices, 'create', Promise.resolve);
+      indicesDelete = sinon.stub(manager.client.indices, 'delete', Promise.resolve);
+    });
+
+    it('should be able to load scenarios', function (done) {
+      manager.load('makelogs')
+      .then(function () {
+        expect(create.getCall(0).args[0].index).to.be('logstash-2015.09.17');
+        expect(create.getCall(1).args[0].index).to.be('logstash-2015.09.18');
+        expect(bulk.called).to.be(true);
+        done();
```

Comment:
Keep in mind that mocha is promise compatible, so you can just return the promise on 23 and skip the `done()` call. This is important because if an error is thrown here by `expect()` it wouldn't be passed to `done()` and reported to the cli